src/test/run-pass-fulldeps/auxiliary/issue-40001-plugin.rs

```diff
@@ -26,20 +26,8 @@ pub fn plugin_registrar(reg: &mut Registry) {
     reg.register_attribute("whitelisted_attr".to_string(), Whitelisted);
 }
 
-declare_lint!(MISSING_WHITELISTED_ATTR, Deny,
-              "Checks for missing `whitelisted_attr` attribute");
-
-struct MissingWhitelistedAttrPass;
-
-impl LintPass for MissingWhitelistedAttrPass {
-    fn name(&self) -> &'static str {
-        "MissingWhitelistedAttrPass"
-    }
-
-    fn get_lints(&self) -> LintArray {
-        lint_array!(MISSING_WHITELISTED_ATTR)
-    }
-}
+declare_lint!(MISSING_WHITELISTED_ATTR, Deny,"Checks for missing `whitelisted_attr` attribute");
```

Comment:
```suggestion
declare_lint!(MISSING_WHITELISTED_ATTR, Deny, "Checks for missing `whitelisted_attr` attribute");
```

irrelevant with reversion to the multi-line version of `declare_lint`